### ISSUE TITLE
Don't annoy GC

### ISSUE THREAD

**Issue by JulienBe** [julienvillegas/libgdx.info-Shader-Shockwave#2]
https://github.com/julienvillegas/libgdx.info-Shader-Shockwave/blob/f139995adac7f5fcea3d301144f1d75260e49dff/core/src/com/mygdx/game/ShockWave.java#L91

I think that there is a new that shoudln't be there :smile_cat:

**Comment by gderuki** [comment]
Agreed, you could save lots of allocations by moving `new` to `start()`
